frontend/src/metabase/dashboard/actions/cards-typed.ts

```diff
@@ -154,7 +154,7 @@ export const addCardToDashboard =
         tabId,
         dashcardOverrides: { id: dashcardId, card, card_id: cardId },
       }),
-    );
+    ) as DashboardCard;
```

Comment:
couldn't we use `dashcard: DashboardCard`?

Reply:
the issue is the `dispatch` type - no matter what, it's returning `unknown`. I tried to switch it to the redux toolkit version of the type but it just caused more errors